CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All changes to the images, whether it's a new image or an update will be added h
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 
+## 2017-08-21
```

Comment:
Shall we move the changes to be under the UNRELEASED section since they haven't been rolled to production yet?

Reply:
Thanks, done!